tests/steps/facebook.coffee

```diff
@@ -1,6 +1,6 @@
 module.exports = ->
   @Given /^I create a Facebook Share Button$/, () ->
-    @driver.visit("http://localhost:8000/tests/fixtures/facebook.html")
+    @driver.visit("file:///#{process.cwd()}/tests/fixtures/facebook.html")
```

Comment:
you just need to run `python -m SimpleHTTPServer` before you run the tests (this is how it's handled on travis-ci as well)